src/modules/validation_manager.cpp

```diff
@@ -0,0 +1,115 @@
+#include "die.h"
+#include "validation_manager.h"
+#include "config/board_config.h"
+#include "nrf_log.h"
+
+#include "animations/animation.h"
+#include "modules/anim_controller.h"
+#include "animations/animation_name.h"
+
+#include "bluetooth/bluetooth_messages.h"
+#include "bluetooth/bluetooth_message_service.h"
+#include "bluetooth/bluetooth_stack.h"
+
+#include "drivers_nrf/power_manager.h"
+#include "data_set/data_set.h"
+
+#include "utils/Utils.h"
+
+using namespace Animations;
+using namespace Modules;
+using namespace Config;
+using namespace DriversNRF;
+using namespace Bluetooth;
+
+#if !defined(FIRMWARE_VERSION)
+    #warning Firmware version not defined
+    #define FIRMWARE_VERSION "Unknown"
+#endif
+
+namespace Modules::ValidManager
+{
+	static AnimationName nameAnim;
+	bool isPlaying;
+
+	void skipFeed(void* token);
+	void stopNameAnim();
+	void startNameAnim();
+	void onConnection(void* token, bool connected);
+    void exitValidationMode(void* token, const Message* msg);
+    void WhoAreYouHandler(void* token, const Message* message);
+    void EnterSleepMode(void* token, const Message* msg);
+
+	// Initializes validation animation objects and hooks AnimController callback
+	void init() 
+	{
+		isPlaying = false;
+
+		// Name animation object
+        nameAnim.type = Animation_Name;
+		nameAnim.framesPerBlink = 3; // 3 animation frames per blink
+		nameAnim.setDuration(1000);
+		nameAnim.brightness = 0x10;
+
+		// Assign callback for AnimController to skip PowerManager::feed
+		AnimController::hook(skipFeed, nullptr);
```

Comment:
This isn't needed, it's nice to have a comment explaining why we wan' to skip calling PowerManager::feed though.
However this should be in die_init.cpp as this where the hook is setup in other modes